src/lib/utils/arithmetic_operator_expression.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "arithmetic_operator_expression_impl.hpp"
+
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * Returns the std::function for and ExpressionType that is an arithmetic operator with both operands being the same type.
```

Comment:
I don't understand this sentence.

Reply:
Updated the comment to: `Returns the arithmetic operator function equivalent to the given ExpressionType.`